rllib/algorithms/impala/impala.py

```diff
@@ -584,7 +584,10 @@ def training_step(self) -> ResultDict:
         # Aggregation workers are stateless, so we do not need to restore any
         # state here.
         if self._aggregator_actor_manager:
-            self._aggregator_actor_manager.probe_unhealthy_actors()
+            self._aggregator_actor_manager.probe_unhealthy_actors(
```

Comment:
**q**: OOC, did you catch these in the impala's ci tests?

Reply:
huh, no. that would require us to create a way to mock out these aggregate workers too.
kind of too much for this PR?

Reply:
no I mean does impala's unittests fail if you don't set `mark_healthy=True`?